Example/Example/EditorDemoController.swift

```diff
@@ -244,29 +230,12 @@ class EditorDemoController: UIViewController {
     // MARK: - Title and Title placeholder position methods
     func updateTitlePosition() {
         let referenceView: UIScrollView = editingMode == .richText ? richTextView : htmlTextView
-        titleTopConstraint.constant = -(referenceView.contentOffset.y+referenceView.contentInset.top)
+        titleTopConstraint.constant = -(referenceView.contentOffset.y + referenceView.contentInset.top) + 5
```

Comment:
Mind moving the magic numbers to constants? perhaps something like this would become handy (in the `struct Constants`):

```
let titleInsets = UIEdgeInsets(top: 5, left: 0, bottom: 5, right: 0)
```